### PR TITLE
Sidebar status icons should have accessibility attributes

### PR DESCRIPTION
**Pull request by allanthompson-tpr**
#### Fixes [AB#82763](https://dev.azure.com/thepensionsregulator/0158f35d-cea5-4d9d-adc8-9fc3d3d98793/_workitems/edit/82763)

The sidebar and card status icons should have accessibility attributes such that screen reader users can understand the information being conveyed by the icons displayed.

### CODE REVIEW

**Review by sussexrick — COMMENTED** [review]
If it's possible to move the image inside the `<Link>` I think that would be helpful. That would mean when you tabbed to a link a screenreader would say "Link Asset Breakdown Section Not Complete" rather than "Link Asset Breakdown" and then wait for the user to discover that there's an image afterwards with further information.

**Comment by allanthompson-tpr** [comment]
> If it's possible to move the image inside the `<Link>` I think that would be helpful. That would mean when you tabbed to a link a screenreader would say "Link Asset Breakdown Section Not Complete" rather than "Link Asset Breakdown" and then wait for the user to discover that there's an image afterwards with further information.

This has been done in commit 77b58b9, I've tested using NVDA in Chrome, FIrefox & IE11 and in each of those browsers NVDA reads out the link name with the status as suggested.

In Edge, NVDA does not read out the status, but I'm not sure there's anything we could do about that.

File: packages/layout/src/components/__tests__/sidebar.spec.tsx [hunk 1]
```diff
@@ -160,4 +170,44 @@ describe('Sidebar', () => {
 		expect(totalCompleted.length).toMatchInlineSnapshot(`3`);
 		expect(totalSections.length).toMatchInlineSnapshot(`8`);
 	});
+
+	test('each section status icon hass accessibility attributes', () => {
```
Comment (sussexrick) [inline packages/layout/src/components/__tests__/sidebar.spec.tsx:174]: Typo here on "hass"

**Review by sussexrick — APPROVED** [review]

File: packages/layout/src/components/__tests__/sidebar.spec.tsx (hunk 1 above)
Comment (allanthompson-tpr) [reply packages/layout/src/components/__tests__/sidebar.spec.tsx]: Typo fixed.

**Pull request merged**